project/plugins.sbt

```diff
@@ -1,3 +1,4 @@
 libraryDependencies ++= Seq("com.github.pathikrit" %% "better-files" % "3.6.0")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.6")
\ No newline at end of file
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.6")
```

Comment:
You can remove this plugin since it is already imported by codacy-sbt-plugin

Reply:
Done ✅